src/main/java/org/terasology/dynamicCities/rasterizer/parts/RectPartRasterizer.java

```diff
@@ -47,7 +47,7 @@ protected void raster(RasterTarget brush, RectBuildingPart part, HeightMap heigh
         }
 
 //        int topHeight = part.getBaseHeight() + part.getWallHeight() + part.getRoof().getHeight;
-//        Region3i bbox = Region3i(rc.minX(), part.getBaseHeight(), rc.minY(), rc.maxX(), topHeight, rc.maxY());
+//        BlockRegion bbox = new BlockRegion(rc.minX(), part.getBaseHeight(), rc.minY(), rc.maxX(), topHeight, rc.maxY());
 
 //        if (chunk.getRegion().overlaps(bbox)) {
```

Comment:
can we just remove this code 🙄

Reply:
this PR was intended to only get rid of Region3i stuff... I don't know what this was used for and whether or not it is still relevant.